hazelcast-client/src/main/java/com/hazelcast/client/map/impl/nearcache/ClientHeapNearCache.java

```diff
@@ -90,21 +90,21 @@ public ClientHeapNearCache(String mapName, ClientContext context, NearCacheConfi
         this.timeToLiveMillis = TimeUnit.SECONDS.toMillis(nearCacheConfig.getTimeToLiveSeconds());
         this.invalidateOnChange = nearCacheConfig.isInvalidateOnChange();
         this.evictionPolicy = EvictionPolicy.valueOf(nearCacheConfig.getEvictionPolicy());
-        this.canCleanUp = new AtomicBoolean(true);
+        this.canExpire = new AtomicBoolean(true);
```

Comment:
what about renaming this field 'expireInProgress' since it is uses as a lock to prevent concurrent cleanups.